app/server/src/main/resources/reference.conf

```diff
@@ -1,3 +1,62 @@
+bitcoin-s {
+    datadir = ${HOME}/.bitcoin-s
+    network = regtest # regtest, testnet3, mainnet
+
+    logging {
+        level = WARN # trace, debug, info, warn, error, off
+
+        # You can also tune specific module loggers.
+        # They each take the same levels as above.
+        # If they are commented out (as they are
+        # by default), `logging.level` gets used
+        # instead.
+        # The available loggers are:
+
+        # incoming and outgoing P2P messages
+        # p2p = info
+
+        # verification of block headers, merkle trees
+        # chain-verification = info
+
+        # generation of addresses, signing of TXs
+        # key-handling = info
+
+        # wallet operations not related to key management
+        # wallet = info
+
+        # HTTP RPC server
+        # http = info
+
+        # Database interactions
+        # database = info
+
+        # whether or not to write to the log file
+        disable-file = false
+
+        # whether or not to log to stdout
+        disable-console = false
+    }
+
+    node {
+        peers = ["localhost"]
```

Comment:
can we you put examples here for things you can specify, for instance i would expect `testnet.neutrino.suredbits.com:18333` to work as well